modules/json/src/json-loader.js

```diff
@@ -25,8 +25,8 @@ export const JSONLoader = {
   */
   category: 'table',
   parse: async (arrayBuffer, options) =>
```

Comment:
This function is synchronous.

Reply:
Yes... let me elaborate on my answer to your previous comment around this:
- the loader object `parse` function is defined to be asynchronous in the loader object docs, and every loader is expected to offer this function. Applications can use loaders without importing loaders.gl core because of this.
- This loader object is also offering a `parseTextSync` so `parseSync(..., JSONLoader)` will work.